ElasticBox/user-authentication.md

```diff
@@ -27,17 +45,43 @@ Add LDAP sources in ElasticBox to match the structure of LDAP in your organizati
 When users sign in to ElasticBox with their LDAP credentials, we don’t store their passwords. The login session passes on their credentials to each LDAP source defined in ElasticBox. The LDAP server looks the user up by their username or Use Principal Name (UPN), typically in the **yourname@example.com** format. The server responds with an authorized or unauthorized request. If authorized, we grant the user access in ElasticBox. Else, we deny access.
 
 **In this article:**
+<<<<<<< HEAD
+<<<<<<< HEAD
+* Setting up LDAP in ElasticBox
+* Syncing with LDAP groups
+* Giving LDAP accounts admin access
+=======
+* [Setting up LDAP in ElasticBox](../ElasticBox/user-authentication.md)
+* [Syncing with LDAP groups](../ElasticBox/user-authentication.md)
+* [Giving LDAP accounts admin access](../ElasticBox/user-authentication.md)
+>>>>>>> CenturyLinkCloud/master
+=======
 * [Setting up LDAP in ElasticBox](../ElasticBox/user-authentication.md)
 * [Syncing with LDAP groups](../ElasticBox/user-authentication.md)
 * [Giving LDAP accounts admin access](../ElasticBox/user-authentication.md)
+>>>>>>> CenturyLinkCloud/master
 
 ### Setting Up LDAP in ElasticBox
 
 **Steps**
+<<<<<<< HEAD
```

Comment:
Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.